package-lock.json

```diff
@@ -2394,24 +2394,24 @@
             }
         },
         "node_modules/dockerfile-language-service": {
-            "version": "0.9.0",
-            "resolved": "https://registry.npmjs.org/dockerfile-language-service/-/dockerfile-language-service-0.9.0.tgz",
-            "integrity": "sha512-sDUkTR4LUimEAWXDIbUTEx2CytCBlx+XlJkg4B2Ptvak9HkwPD4JpXesaWxXPpp6YHCzxqwsTDY7Gf79ic340g==",
+            "version": "0.10.0",
+            "resolved": "https://registry.npmjs.org/dockerfile-language-service/-/dockerfile-language-service-0.10.0.tgz",
+            "integrity": "sha512-1EfLBQViRSzRFFet8co2r3CVudniKPbDB/Q595RUouewUs6UKpe4pdzrTuXIc4TzIGdRXUnBZCZdiN0T9fl+8w==",
             "dependencies": {
                 "dockerfile-ast": "0.4.2",
```

Comment:
Should `dockerfile-language-service` be updated to use `dockerfile-ast@0.5.0`, to match what is used by `dockerfile-utils@0.11.0`?

Reply:
Good call. I have made the change and released a new 0.10.1 version.